### PR TITLE
Translated more far:config help pages into English.

### PR DESCRIPTION
**Pull request by MKadaner**

### CODE REVIEW

File: far/FarEng.hlf.m4 [hunk 1]
```diff
@@ -6343,92 +6343,91 @@ in HKCU hive is ignored.
...
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @Editor.NewFileUnixEOL
 $ #far:config Editor.NewFileUnixEOL#
- Параметр "Editor.NewFileUnixEOL" позволяет задавать умолчательные символы конца строки в редакторе
-для вновь создаваемых файлов.
-
- false - ^<wrap>Во вновь создаваемых файлах строки заканчиваются символами #<CR><LF># - (как в Windows).
+ This Boolean parameter specifies line break style in the files created
```
Comment (alabuzhev) [inline far/FarEng.hlf.m4:6376]: [Line endings](https://www.microsoft.com/en-us/language/Search?&searchTerm=line%20endings&langID=635&Source=true&productid=0) maybe?

File: far/FarRus.hlf.m4 [hunk 2]
```diff
@@ -6457,90 +6457,91 @@ HKLM установлен в ноль (на локальной машине не
 
 @Editor.KeepEditorEOL
 $ #far:config Editor.KeepEditorEOL#
- Параметр "Editor.KeepEditorEOL" позволяет управлять типом перевода строк для текста
-вставляемого в редактор из буфера обмена.
+ Этот логический (Boolean) параметр управляет тем, как переводы строк
```
Comment (alabuzhev) [inline far/FarRus.hlf.m4:6460]: [Завершения строк](https://www.microsoft.com/en-us/language/Search?&searchTerm=line%20endings&langID=635&Source=true&productid=0) maybe?

File: far/FarEng.hlf.m4 (hunk 1 above)
Comment (MKadaner) [reply far/FarEng.hlf.m4:6376]: "Line ending" appears in just one place in FarEng.hlf.m4 file, while "line break" is in at least 18 places (likely more). This is why I choose the latter term. I can live with either. If you have strong preference, please let me know, I'll change all places to be consistent.

Comment (alabuzhev) [reply far/FarEng.hlf.m4:6376]: I don't have a strong preference, let's keep it consistent with the rest of the file.

File: far/FarEng.hlf.m4 [hunk 3]
```diff
@@ -6343,92 +6343,91 @@ in HKCU hive is ignored.
...
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @Editor.AddUnicodeBOM
 $ #far:config Editor.AddUnicodeBOM#
- Параметр "Editor.AddUnicodeBOM" указывает надо ли добавлять BOM (Byte Order Mark) в начало создаваемых
-редактором файлов в юникодной кодировке (1200, 1201, 65001).
+ This Boolean parameter specifies whether Byte Order Mark (BOM) is added
+at the beginning of the files created by the Editor and saved in
+a UNICODE encoding (code pages 1200, 1201, 65001).
```
[the remark this answers is not in the document]
Comment (MKadaner) [reply far/FarEng.hlf.m4:6364]: Makes sense. Changed.

File: far/FarEng.hlf.m4 [hunk 4]
```diff
@@ -6343,92 +6343,91 @@ in HKCU hive is ignored.
...
 
 
 @Macros.ShowPlayIndicator
 $ #far:config Macros.ShowPlayIndicator#
- Параметр "Macros.ShowPlayIndicator" позволяет включать или отключать отображение в левом верхнем углу экрана символа '\2FP\-'
-во время воспроизведения макропоследовательности.
-
- false - ^<wrap>Символ не отображается
+ This Boolean parameter turns macro playback indicator (symbol ‘\2FP\-’
+at the left-hand top screen corner) ON or OFF.
```
[the remark this answers is not in the document]
Comment (MKadaner) [reply far/FarEng.hlf.m4:6406]: Thanks, will change. Do not know why all uppercase. Lowered.

File: far/FarEng.hlf.m4 [hunk 5]
```diff
@@ -6343,92 +6343,91 @@ in HKCU hive is ignored.
...
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @Viewer.SearchWrapStop
 $ #far:config Viewer.SearchWrapStop#
- Параметр "Viewer.SearchWrapStop" позволяет изменять поведение при прохождении через начало/конец
-файла при поиске в просмотрщике.
+ This tri-state parameter controls the behavior of continue search key
```
[the remark this answers is not in the document]
Comment (MKadaner) [reply far/FarEng.hlf.m4:6418]: Sure. Thanks.

File: far/FarEng.hlf.m4 (hunk 1 above)
Comment (MKadaner) [reply far/FarEng.hlf.m4:6376]: Changed to `line break` (plus minor cleanup).

File: far/FarRus.hlf.m4 (hunk 2 above)
Comment (MKadaner) [reply far/FarRus.hlf.m4:6460]: Apparently, the phrase `перевод строки` is used consistently in the entire FarRus.hlf.m4. Let's keep it for now. If you think `Завершение строки` (or anything else) is better, please let me know, I'll change it everywhere in a separate PR.

**Comment by MKadaner** [comment]
Ready to merge (if looks good).

**Comment by alabuzhev** [comment]
Thanks

**Pull request merged**